Kitodo-DataManagement/src/main/java/org/kitodo/data/database/beans/LtpValidationConfiguration.java

```diff
@@ -0,0 +1,183 @@
+     * 
+     * @return true if finishing an image validation task should be canceled if an error was found during validation
+     */
+    public boolean getRequireNoErrorToFinishTask() {
+        return this.requireNoErrorToFinishTask;
+    }
+
+    /**
+     * Sets the title of this validation configuration.
+     * 
+     * @param title the title of this validation configuration
+     */
+    public void setTitle(String title) {
+        this.title = title;
+    }
+
+    /**
+     * Sets the mime type this validation configuration can be applied to.
+     * 
+     * @param mimeType the mime type this validation configuration can be  applied to
+     */
+    public void setMimeType(String mimeType) {
+        this.mimeType = mimeType;
+    }
+
+    /**
+     * Sets the list of validation conditions that are checked for an image.
+     * 
+     * @param validationConditions the list of validation conditions that are checked for an image
+     */
+    public void setValidationConditions(List<LtpValidationCondition> validationConditions) {
+        this.validationConditions = validationConditions;
+    }
+
+    /**
+     * Sets the list of folders that use this validation configuration.
+     * 
+     * @param folders the list of folders that use this validation configuration
+     */
+    public void setFolders(List<Folder> folders) {
+        this.folders = folders;
+    }
+
+    /**
+     * Sets whether uploading an image should be canceled if an error was found during validation.
+     * 
+     * @param requireNoErrorToUploadImage true if uploading an image should be canceled
+     */
+    public void setRequireNoErrorToUploadImage(boolean requireNoErrorToUploadImage) {
+        this.requireNoErrorToUploadImage = requireNoErrorToUploadImage;
+    }
+
+    /**
+     * Sets whether finishing an image validation task should be canceled if an error was found during validation.
+     * @param requireNoErrorToFinishTask true if finishing an image validation task should be canceled
+     */
+    public void setRequireNoErrorToFinishTask(boolean requireNoErrorToFinishTask) {
+        this.requireNoErrorToFinishTask = requireNoErrorToFinishTask;
+    }
+
+    /**
+     * Equals implementation based on database id.
+     */
+    @Override
+    public boolean equals(Object object) {
+        if (this == object) {
+            return true;
+        }
+
+        if (object instanceof LtpValidationConfiguration) {
+            LtpValidationConfiguration configuration = (LtpValidationConfiguration) object;
+            return Objects.equals(this.getId(), configuration.getId());
+        }
+
+        return false;
+    }
+
+    /**
+     * Hash code implementation based on all properties of validation configuration.
+     */
+    @Override
+    public int hashCode() {
+        return Objects.hash(id, title, mimeType, requireNoErrorToUploadImage, requireNoErrorToFinishTask, validationConditions);
+    }
```

Comment:
See above, `equals()` and `hashCode()` must be implemented must be implemented with the same comparison logic. **This needs to be changed.**